Gtk-4.0.gir

```diff
@@ -75433,7 +75433,7 @@ current render node of @snapshot.</doc>
       <method name="free_to_node" c:identifier="gtk_snapshot_free_to_node" introspectable="0">
         <doc xml:space="preserve">Returns the node that was constructed by @snapshot
 and frees @snapshot.</doc>
-        <return-value transfer-ownership="full">
+        <return-value transfer-ownership="full" nullable="1">
```

Comment:
This will cause an API change @bilelmoussaoui

Reply:
I already did a release with that change

Reply:
Even better, we can get rid of some overrides